index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <!--head containing links to css and the title-->
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Horiseon Inc.</title>
```

Comment:
As mentioned in the feedback, this could be more descriptive. Consider adding a bit more detail.